src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -155,7 +154,7 @@ private bool IsBranchTarget(ILProcessor processor, Instruction instruction)
             return false;
         }
 
-        private void ReplaceInstructionTarget(Instruction instruction, Instruction oldTarget, Instruction newTarget)
+        private static void ReplaceInstructionTarget(Instruction instruction, Instruction oldTarget, Instruction newTarget)
```

Comment:
Any particular reason you made this `static`?

Reply:
No reason aside from them not relying on any fields in the class. It's not a functional change in behavior, mostly cosmetic.